src/main/resources/resources/logisim/strings/std/std.properties

```diff
@@ -797,6 +797,7 @@ TTL74266 = 74266: quad 2-input XNOR gate (open-collector)
 TTL7427 = 7427: triple 3-input NOR gate
 TTL74273 = 74273: octal D-Flipflop with clear
 TTL74283 = 74283: 4-bit binary full adder
+TTL74299 = 74299: 8-bit universal shift register with 3-state outputs
```

Comment:
"3-state" -> "three-state"

Reply:
Done. I just copied the description straight from the TI datasheet